charts/wharf-helm/templates/web.yaml

```diff
@@ -102,6 +102,23 @@ data:
       },
       "UpdateLatency": 20000,
       "UpdateFrequency": 30000,
+      {{- with .Values.web.oidc }}
```

Comment:
Suggest adding an if-statement on a new field `.Values.web.oidc.enabled` to toggle this entire section, and leave it disabled by default.

Think it'd be wise to have this as an opt-in feature rather than an opt-out. At least to begin with, and to keep backward compatibility.

Such as:

```mustache
      {{- with .Values.web.oidc }}
      {{- if .enabled }}
      // JSON definition goes here
      {{- end }}
      {{- end }}
```

Reply:
Fixed